curta/src/chip/arithmetic/expression.rs

```diff
@@ -5,8 +5,12 @@ use super::expression_slice::ArithmeticExpressionSlice;
 use crate::air::parser::AirParser;
 use crate::chip::register::memory::MemorySlice;
 use crate::math::prelude::*;
+
+// how arithmetic experession translate to this? any example? e.g. a + b - c==0?
```

Comment:
a vector does not mean many diff constraints on the same register.

it's very similar to numpy. a * b -c, where a,b,c are vectors

Reply:
so for example [0, 1, 2] + [1, 1, 1] = [1, 2, 3] is a + b == c.

Reply:
guess my main question is how the vector of polynomial represent a constraint, assume if it's an add arithmetic on some slice of memories (here assume column a + column b = column c) then the arithmetic expression should have 3 polynomial where each represent values of the column a,b, and c? i'm still confused on how this expression connect with the trace.